Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -154,8 +154,11 @@ extension SPSettingsViewController {
 }
 
 private struct AccountDeletion {
+    static func confirmAlertMessage(with email: String) -> String {
+        String(format: confirmAlertTemplate, email)
+    }
     static let deleteAccount = NSLocalizedString("Delete Account", comment: "Delete account title and action")
-    static let confirmAlertMessage = NSLocalizedString("By deleting your account, all notes created with this account will be permanently deleted. This action is not reversible", comment: "Delete account confirmation alert message")
+    static let confirmAlertTemplate = NSLocalizedString("By deleting the account for %@, all notes created with this account will be permanently deleted. This action is not reversible", comment: "Delete account confirmation alert message")
```

Comment:
I am guessing these copy changes being last minute and also post-code freeze, they haven't had a change to be imported in GlotPress, and thus would remain untranslated in the final version – even if the old copy was already translated by then. (Just wanted to make sure you and the developers were aware of that consequence of changing strings post-❄️ )

Reply:
I am aware of this. It was a last minute change so I am guessing they had to do it. Let's cc @charliescheer just in case he wasn't familiar with it.

Reply:
Thanks for the additional details.  I was not aware of this until after I added the update to the strings. I will keep that in mind with any future post freeze updates!